src/Lesson4/Boxer.java

```diff
@@ -0,0 +1,32 @@
+package Lesson4;
+/*
+2. Implement the method fight() depending on weight, strength and age of the boxer:
+You should write the dependency by yourself. The method should return true or false if our boxer won or lost.
+*/
+
+public class Boxer {
+    public int age;
+    public int weight;
+    public int strength;
+
+    public Boxer(int age, int weight, int strength) {
+        this.age = age;
+        this.weight = weight;
+        this.strength = strength;
+    }
+
+    public boolean fight(Boxer anotherBoxer) {
+        if (weight + strength - age > anotherBoxer.weight + anotherBoxer.strength - anotherBoxer.age){
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Could be shorter:
return (weight + strength - age > anotherBoxer.weight + anotherBoxer.strength - anotherBoxer.age);